Source/Core/Core/PowerPC/GDBStub.cpp

```diff
@@ -923,16 +907,11 @@ static void HandleAddBreakpoint()
 
 static void HandleRemoveBreakpoint()
 {
-  u32 type, addr, len, i;
+  u32 type(Hex2char(s_cmd_bfr[1])), addr(0), len(0), i(3);
```

Comment:
Would rather the declaration and assignment of `addr`, `len`, and `i` be moved down to where it is used (where the assignment was previously). Also, why suddenly parenthesis for initialization?

Reply:
I've seen both parentheses and equal signs used for initialization in the codebase and I didn't see any pattern for when to choose which.
I probably edited this part right after some part which used parentheses.